tests/MongoDB.Driver.Tests/Specifications/server-selection/ServerSelectionTests.cs

```diff
@@ -0,0 +1,93 @@
+/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Threading;
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Bson.TestHelpers.XunitExtensions;
+using MongoDB.Driver.Core;
+using MongoDB.Driver.Core.Clusters;
+using MongoDB.Driver.Core.Events;
+using MongoDB.Driver.Core.Misc;
+using MongoDB.Driver.Core.TestHelpers.XunitExtensions;
+using MongoDB.Driver.TestHelpers;
+using Xunit;
+
+namespace MongoDB.Driver.Tests.Specifications.server_selection
+{
+    public class ServerSelectionTest
```

Comment:
This filename and class name are suspicious.

We aren't testing server selection.

We are testing that `CommandUsingCommandMessageWireProtocol` does not send `$readPreference` to standalone servers.

But at the moment a proper name and place to test this escapes me. I just am pretty sure that `ServerSelectionTests` is not right.

Reply:
That name comes from https://github.com/ksadoff/mongo-csharp-driver/pull/9#discussion_r303530255. We also discussed ReadPreferenceTests, but that may be too generic, and the name comes from the fact that the changes for this ticket originated int the server-selection part of the spec repo. But I'd be happy to change it to something more appropriate.

Reply:
I don't think the ServerSelectionTests name is acceptable. That is the name that would be given to a file containing server selection tests.

Also, the location of this file isn't right. It's currently sitting in the `...\Specifications\server-selection` folder but not only are these tests not server selection related they also aren't JSON driven tests from a specification.

Reply:
Changed to `ReadPreferenceOnStandaloneTests.cs` and moved to `MongoDB.Driver.Tests` per offline discussion with @rstam